README.md

```diff
@@ -10,6 +10,7 @@ Enables Jenkins to manage and upload Android app files (AAB or APK) to Google Pl
 - Uploading Android App Bundle (AAB) or APK files to Google Play
   - This includes apps which use Multiple APK support
   - ProGuard `mapping.txt` files can also be associated with each app file, for deobfuscating stacktraces
+  - Native debug symbol `lib.zip` files can also be associated with each app file, for deobfuscating native crash dumps
```

Comment:
Maybe it would be worth noting in the README, that if using gradle 4.1 with the option `ndk.debugSymbolLevel="FULL|SYMBOL_TABLE"`, this upload is unnecessary and may have unexpected results.

Reply:
See my comment about adding the help-*.html resource, which Jenkins will display automatically when you click the (?) icon for the field. That would be a more appropriate place for it.